stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/StreamFileUtil.kt

```diff
@@ -26,6 +26,7 @@ import io.getstream.chat.android.core.internal.InternalStreamChatApi
 import java.io.File
 import java.io.IOException
 
+private const val DEFAULT_QUALITY = 90
```

Comment:
Should this be PascalCase? Like the rest of Kotlin constants

Reply:
PascalCase is only for compose modules, as it is the default style from jetpack compose